package.json

```diff
@@ -9,14 +9,15 @@
     "build": "tsc -p tsconfig.build.json",
     "dev": "nodemon src/index.ts",
     "start": "node dist/src/index.js",
```

Comment:
We should add "generate" script within start otherwise build or deployment would fail.